src/declarePropsNonNullish.tsx

```diff
@@ -0,0 +1,15 @@
+type DeclarePropsNonNullish = <TProps, TPropNames extends keyof TProps>(
+  propNames: TPropNames | TPropNames[],
```

Comment:
Per [my comment](https://github.com/sage-health/intake-and-application-api/pull/171/files/6dce02ebd453a12208af9582f4addb565d3e0489#r448363841), I played with changing the signature of `declarePropsNonNullish()` to be eg `declarePropsNonNullish<'someProp'>()` instead of `declarePropsNonNullish(['someProp'])`, but the typing didn't seem as strong and I think I prefer the latter, so kept as-is